tests/filecheck/mlir-conversion/with-mlir/dialects/irdl/testd.irdl.mlir

```diff
@@ -1,3 +1,4 @@
+// XFAIL: *
```

Comment:
:thinking:

Reply:
Oh good catch, I guess I forgot a regression!

So, irdl as a dialect evolved on MLIR's side meanwhile and need some more-than-switching-to-properties work on xDSL's side to work.
This XFAIL was decided with @math-fehr, and it says "this test is expected to fail".

Reply:
I'm fine leaving it for now, I'll update this later on.
This is mostly because this change cannot be added incrementally from what I understand.
Because we need to update the dialect, but for that we need to update MLIR at the same time.
So let's update it after we merge this change, so we make this more incremental? (It's not like anyone cares about that dialect now besides me anyway)